docs/internals/protocol.rst

```diff
@@ -292,6 +292,8 @@ format:
 
 .. code-block:: javascript
 
+    // XXX: No longer applicable?
```

Comment:
The response is still the same but the ping task is elsewhere.

Reply:
Not sure I understand... is the response _exactly_ the same, or is it a same-except-the-`"task"`-value-changes sort of sameness? If the former I'll just delete my comment, if the latter... perhaps someone can supply an updated JSON response snippet?